rust-toolchain

```diff
@@ -1 +1 @@
-1.56.1
+nightly-2021-11-17
```

Comment:
These two:
`1.62.0-nightly (bed05e996 2022-05-02) -> 1.63.0-nightly (546c826f0 2022-06-12)` are newer. If clippy works for any of them, we could maybe update?

Reply:
Updated with `2022-05-02`